modules/network-observability-release-notes-1-3-0-advisory.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+// * network_observability/release_notes_archive/network-observability-operator-release-notes-archive.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-release-notes-1-3-0-advisory_{context}"]
+== Network observability 1.3.0 advisory
```

Comment:
```suggestion
= Network observability 1.3.0 advisory
```
needs to be an L1; kinda surprised that didn't throw a Vale error (I'll look into that)